htmlify.js

```diff
@@ -1,20 +1,41 @@
-// setup mutation observer, because this is the new cool way to see if elements are mounted
```

Comment:
:open_mouth: oh no! What's the reason for removing this comment?

Reply:
I thought you mentioned it because you were figuring it out when you developed. It doesn't add much value to the code so I removed it.